ewkcoffea/modules/corrections.py

```diff
@@ -643,3 +635,34 @@ def ApplyJetVetoMaps(jets,year):
     #Sum the outputs for each event (if the sum is >0, the event will fail)
     veto_map_event = ak.sum(jet_vetomap_score, axis=-1)
     return veto_map_event
+
```

Comment:
Please add a comment to explain what this function does. It can be short, but just something to start from would be great (image a new grad student reading the code, what is one line you would want them to know to have an idea of what this function does)